index.html

```diff
@@ -604,7 +604,7 @@ <h4>NewPipe is free and open source</h4>
                         </ul>
                         <p class="button-wrapper">
                             <a href="https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md#newpipe-contribution-guidelines"><button class="action">Contribute</button></a>
-                            <a href="https://github.com/TeamNewPipe/NewPipe/"><button class="action"><i class="fa fa-star"></i> <span class="api-stargazers">3000</span></button></a>
+                            <a href="https://github.com/TeamNewPipe/NewPipe/"><button class="action"><i class="fa fa-star"></i> <span class="api-stargazers">7000</span></button></a>
```

Comment:
Another of these data we fetch live. But this value doesn't change so often, and is unlikely to go down suddenly. Perhaps append a little `+` to kind of show it's a potentially old value?

Reply:
I will definitly add a little + for you 😘 
That fetching also didnt work on my local instance :(

Reply:
Hmm... might be a cross-origin problem? We're not sending any CORS headers there. Mind to open a new issue?

Reply:
I should be able to see this in the webconsole then right?

Reply:
See what? That there's some CORS issue? Depends on your browser, normally there should be some sort of error message somewhere.

Reply:
Yup, there is, see #129

Reply:
This is btw still open since we havent decided how to handle that case in the other comment, I personally like the + idea, so even people with JS disabled can see that content.